src/services/datahub-client.ts

```diff
@@ -106,6 +106,7 @@ export class HttpDatahubClient implements DatahubClient {
       `domains/${domainName}/events/summarized-by-page`,
       {
         startDate: dateFrom.toISOString(),
+        sortBy: 'visitors',
```

Comment:
@leoslopez, @fcoronelmakingsense,  is it enough to ask for the _visitors_? is the result still in the count property?

![image](https://user-images.githubusercontent.com/1157864/60460853-3cd2aa80-9c1b-11e9-9814-bb3c12c13cd5.png)

![image](https://user-images.githubusercontent.com/8861133/60460370-eca71880-9c19-11e9-9dca-f7cb3103300d.png)

Reply:
sorry my bad. Its visitorsQuantity

Reply:
i fixed now